src/VisualStudio/Core/Def/Implementation/Preview/PreviewUpdater.PreviewDialogWorkspace.cs

```diff
@@ -4,14 +4,14 @@
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.Editor.Shared.Preview;
-using Microsoft.CodeAnalysis.Shared.Extensions;
 using Microsoft.CodeAnalysis.Text;
 
 namespace Microsoft.VisualStudio.LanguageServices.Implementation.Preview
 {
     internal partial class PreviewUpdater
     {
-        private class PreviewDialogWorkspace : PreviewWorkspace
+        // internal for testing
```

Comment:
@jaredpar "internal for testing" comments are a convention we use in lots of places to describe this situation where we really want something to be private and inaccessible to other production code, but we've made it internal for test purposes. Maybe that's not the best approach to this problem, but I'd rather add the comment to make this case consistent with the other cases and separately discuss the larger pattern.